src/js/common/actions/index.js

```diff
@@ -0,0 +1,7 @@
+export const LOG_OUT = 'LOG_OUT';
```

Comment:
This is used by both `login` and `user-profile`. The import was failing in the `user-profile` reducer so I moved it back into `common/actions` (where it used to be until recently)